### ISSUE TITLE
Gun DMG problem

### ISSUE THREAD

**Issue by Tentosu** [SunwellWoW/Sunwell-TBC-Bugtracker#4302]
Description: Hi, I wrote 2 tickets in game and for the last one I was asked to create this ticket.
I have a problem with GUN dmg. All stats on my new gun is better, bud counted DMG in my character is lower.

How it works: I take my new gun and my DMG is lower

How it should work:  New gun bring me more DMG when I equiped it

Source (you should point out proofs of your report, please give us some source): 
There are my screenshots: https://drive.google.com/drive/folders/1ISQRMTein4e-a7-UPFDsnEvOwYE2xeNE?usp=sharing

Look at pictures, one time is with stats, second is with ranged DMG. As you can see my new EPIC have better stats (all of them), bud DMG is lower.

Thank you in advance for any help,
Tentosak

**Comment by firngrod** [comment]
The new gun is faster than the old gun.  While the new gun in at of itself does more damage, your attack power adds DPS to your attacks, meaning that the slower the weapon, the higher the individual attack damage added from your attack power.  This is why the listed damage is lower.  You can still expect your DPS to be higher since you will be shooting more often, not accounting for skills.

Now, because some skills and abilities have damage scaled off attack damage and cooldowns independent of attack speed, the slower gun might still be better for you, but I cannot say if that is the case.

I really thought this kind of stuff was in all the guides for all the classes at this patch.

TL;DR:
Works as intended.

**Comment by Ironjudgment** [comment]
Agree with @firngrod
